app/models/sample.rb

```diff
@@ -466,7 +470,7 @@ def kickoff_pipeline
     # The subsample field of "sample" is currently used as a simple flag (UI checkbox),
     # but was made an integer type in case we want to allow users to enter the desired number
     # of reads to susbample to in the future
-    pr.pipeline_branch = pipeline_branch.blank? ? "master" : pipeline_branch
+    pr.pipeline_branch = pipeline_branch.blank? ? "meera-add-amr" : pipeline_branch
```

Comment:
this is dangerous. please change back to master.

Reply:
resolved